tests/scripts/dind-cluster.sh

```diff
@@ -202,7 +207,8 @@ function dind::run-build-command {
         kube::build::start_rsyncd_container
 
         local rsync_extra=""
-        if (( ${KUBE_VERBOSE} >= 6 )); then
+        if (( KUBE_VERBOSE >= 6 )); then
+            # shellcheck disable=SC2034 # Var is exported
             rsync_extra="-iv"
         fi
```

Comment:
rsync_extra is unused? Then remove it. Or add it to the later `kube::build::rsync` command like it was probably intended. TBH, I'm not sure we use the `dind-cluster.sh` much, so I'm fine just removing it for ease.